ydb/core/tx/datashard/ut_common/datashard_ut_common.cpp

```diff
@@ -2459,6 +2459,73 @@ namespace {
         TDeque<TPendingRequest> QuotaRequests;
     };
 
+    std::unique_ptr<TEvDataShard::TEvReadResult> ReadAllTable(
+        Tests::TServer::TPtr server,
+        ui64 tabletId,
```

Comment:
If we read the entire table than there should be array of tabletIds.

Reply:
@lex007in just rename it to SendReadPart